src/ValueRepresentation.js

```diff
@@ -79,6 +79,7 @@ class ValueRepresentation {
         this._allowMultiple =
             !this._isBinary && singleVRs.indexOf(this.type) == -1;
         this._isExplicit = explicitVRs.indexOf(this.type) != -1;
+        this._storeRaw = true;
```

Comment:
I was wondering if this should be false in the general case to save space.  It seems to me that a very common use case is to read and operate on the data without planning to store it out again.  But unless I'm missing something this overhead is going to be very small compared to the PixelData it's really in the noise level so I'm okay with this being true by default.

Reply:
I'd be fine making it a optional `readOption` if you prefer. But yes space impact should be minimal as it's disabled for binary VRs and sequences.